rmgpy/rmg/model.py

```diff
@@ -795,7 +798,35 @@ def processNewReactions(self, newReactions, newSpecies, pdepNetwork=None):
                         self.core.reactions.remove(rxn)
                     if rxn in self.edge.reactions:
                         self.edge.reactions.remove(rxn)
-
+            
+            if not numpy.isinf(self.toleranceThermoKeepSpeciesInEdge) and spcs != []: #do thermodynamic filtering
+            
+                Tmax = self.Tmax
+                for spc in spcs:
+                    G = spc.thermo.getFreeEnergy(Tmax)
+                    if G > self.Gfmax:
+                        Gn = (G-self.Gmax)/(self.Gmax-self.Gmin)
+                        logging.info('Removing species {0} from edge because it\'s Gibbs number {1} is greater than the toleranceThermoKeepSpeciesInEdge of {2} '.format(spc,Gn,self.toleranceThermoKeepSpeciesInEdge))
```

Comment:
I think it might be more useful to print the actual Gibbs free energy and the Gfmax values. Do you think this should belong in the regular RMG mode or only `debug` mode? Does the other pruning print out each reaction removed in regular mode?